src/game_api/state_structs.hpp

```diff
@@ -16,10 +16,18 @@ struct LightParams
 
 struct Illumination
 {
-    LightParams light1;
-    LightParams light2;
-    LightParams light3;
-    LightParams light4;
+    union
+    {
+        std::array<LightParams, 4> light;
```

Comment:
Should probably be `lights` plural

Reply:
And should probably also document that it's the same as [light1, light2...] since you exposed it to Lua